tools/fiptool/fiptool.h

```diff
@@ -10,8 +10,8 @@
 #include <stddef.h>
 #include <stdint.h>
 
-#include "firmware_image_package.h"
-#include "uuid.h"
+#include <firmware_image_package.h>
```

Comment:
Please move these includes to be in alphabetical order with the includes at lines 10 and 11.

Also, please update the copyright date to 2016-2017.